### DISCUSSION TITLE
Switch to ReadableRegex

### DISCUSSION THREAD

**Discussion by stelmo** [LCSB-BioCore/COBREXA.jl#630] [Ideas] [answered]
We should use [ReadableRegex](https://github.com/jkrumbiegel/ReadableRegex.jl) wherever we use regex in the code. It is simpler to read and probably more maintainable when we are no longer active contributors one day, see [here ](https://github.com/LCSB-BioCore/COBREXA.jl/blob/38a488848258cce832e56d681a9b15974125a1e2/src/base/utils/gene_associations.jl#L65) as a starting point...

**Comment by exaexa** [comment, marked as answer]
We'll switch to actual parsers instead. (after Pikaparser stabilizes a bit)
